pages/desktop/base.py

```diff
@@ -73,9 +73,9 @@ class Footer(Page):
                 'url_suffix': '/about/partnerships/',
             }, {
                 'locator': (By.CSS_SELECTOR, 'ul.links-join > li:nth-of-type(3) >a'),
-                'url_suffix': ('//sendto.mozilla.org/page/contribute/givenow-seq?'
-                               'preset=2&source=mozillaorg_footer&ref=EOYFR2014&utm_campaign=EOYFR2014'
-                               '&utm_source=mozilla.org&utm_medium=referral&utm_content=mozillaorg_footer'),
+                'url_suffix': ('//donate.mozilla.org/?presets=100,50,25,15'
+                              '&amount=50&ref=EOYFR2015&utm_campaign=EOYFR2015'
```

Comment:
flake8 might complain about the lost whitespace that used to line the text up after the paren